src/Tes.Runner/Transfer/MetricsFormatter.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Tes.Runner
+{
+    internal class MetricsFormatter
```

Comment:
Is the purpose of this class to create  a metrics file with the output of the upload/download? What is the difference from getting this information from the standard out - which we will handle in the future?  It seems like this to be a bit out of scope of the current issue.

Reply:
We need to maintain the current metrics file while using this binary to replace blobxfer, which makes it very much a part of this issue. This class accomplishes that. In the future you mention, this class can then go away since it's functionality will be replaced by that future solution.

Reply:
Sounds good.